js/components/directionControl.js

```diff
@@ -0,0 +1,40 @@
+var React = require('react');
+var classnames = require('classnames');
+
+var DirectionControl = React.createClass({
+  getInitialState: function() {
+    return {
+      isTouched: false,
+    };
+  },
+
+  handleEvent: function (ev) {
+    var rotate = ev.type == 'mousedown' || ev.type == 'touchstart';
+    this.props.handleDirection(rotate, this.props.dir);
+    
+    this.setState({
+      isTouched: rotate
+    });
+  },
+
+  render: function () {
+    var directionClass = 'oo-vr-icon--move--' + this.props.dir
+      , touchedDirectionClass = this.state.isTouched && 'oo-vr-icon--move--' + this.props.dir + '--touched';
+    return (
+      <div
+        className={classnames('oo-vr-icon--move direction-control', directionClass, touchedDirectionClass)}
+        key={this.props.dir}
+        tabIndex="0"
```

Comment:
I think this was discussed in the other PR. If this is not meant to be controlled via keyboard, we should remove the tabindex in order to prevent it from getting keyboard focus.

Reply:
this is fixed